Runtime/ScreenshotOptions.cs

```diff
@@ -11,17 +13,9 @@ namespace TestHelper.Monkey
     public class ScreenshotOptions
     {
         /// <summary>
-        /// Directory path to save screenshots.
-        /// Default save path is <c>Application.persistentDataPath</c> + "/TestHelper.Monkey/Screenshots/".
+        /// Strategy for file paths of screenshot images.
         /// </summary>
-        public string Directory { get; set; } = null;
-
-        /// <summary>
-        /// Prefix of screenshots filename.
-        /// Default prefix is <c>CurrentTest.Name</c> when run in test-framework context.
-        /// Using caller method name when run in runtime context.
-        /// </summary>
-        public string FilenamePrefix { get; set; } = null;
+        public Func<FilePath> FilePathStrategy { get; set; } = CounterBasedStrategy.Create(Counter.Global);
```

Comment:
Not `Func`,  Isn't `IScreenshotFilenameStrategy` (or `IScreenshotFilenameBuilder`) preferable?
Can include the implementation of the counter (can be static).

Reply:
I think the return value should be filename (single string) instead of FilePath.
And keep the Directory property.

It is assumed that the following specify allowed. (may need to be rename properties)
directory = /path/to/screenshots
filename = namespace/fixture/testname_n.png